lib/awful/screen.lua

```diff
@@ -234,6 +235,63 @@ function screen.object.set_padding(self, padding)
     end
 end
 
+--- A list of outputs for this screen with their size in mm.
+--
+-- This information is only available when RANDR is used to manage the screens.
+--
+-- Please note that the table content may vary.
+--
+-- An easy way to check if a screen is the laptop screen is usually:
+--
+--    if s.outputs["LVDS-1"] then
+--        -- do something
+--    end
+--
+-- **Signal:**
+--
+--  * *property::outputs*
+--
+-- **Immutable:** true
+-- @property outputs
+-- @param table
+-- @tfield table table.name A table with the screen name as key (like `eDP1` on a laptop)
+-- @tfield integer table.mm_width The screen physical width
+-- @tfield integer table.mm_height The screen physical height
+-- @tfield integer table.name The screen physical height
```

Comment:
...what?!? Could you please rename this to something other than `name` then?

Reply:
/me copy paste facepalm

Reply:
done